applications/NXazint1d.nxdl.xml

```diff
@@ -0,0 +1,298 @@
+        </enumeration>
+      </attribute>
+      <attribute name="signal" type="NX_CHAR">
+        <enumeration>
+          <item value="I"></item>
+        </enumeration>
+      </attribute>
+
+      <field name="I" type="NX_NUMBER">
+        <dimensions rank="2">
+          <dim index="1" value="nImg" />
+          <dim index="2" value="nRad" />
+        </dimensions>
+        <attribute name="long_name" type="NX_CHAR">
+          <enumeration>
+            <item value="intensity"></item>
+          </enumeration>
+        </attribute>
+        <attribute name="units" type="NX_CHAR">
+          <enumeration>
+            <item value="arbitrary units"></item>
+          </enumeration>
+        </attribute>
+      </field>
+
+      <field name="I_errors" type="NX_NUMBER" optional="true">
+        <dimensions rank="2">
+          <dim index="1" value="nImg" />
+          <dim index="2" value="nRad" />
+        </dimensions>
+        <attribute name="long_name" type="NX_CHAR" optional="true">
+          <enumeration>
+            <item value="estimated intensity error"></item>
+          </enumeration>
+        </attribute>
+        <attribute name="units" type="NX_CHAR" optional="true">
+          <enumeration>
+            <item value="arbitrary units"></item>
+          </enumeration>
+        </attribute>
+      </field>
+
+      <field name="radial_axis" type="NX_NUMBER">
+        <dimensions rank="1">
+          <dim index="1" value="nRad" />
+        </dimensions>
+        <attribute name="long_name" type="NX_CHAR">
+          <enumeration>
+            <item value="q"></item>
+            <item value="2theta"></item>
+          </enumeration>
+        </attribute>
+        <attribute name="units" type="NX_CHAR">
+          <enumeration>
+            <item value="NX_PER_LENGTH"></item>
+            <item value="NX_WAVENUMBER"></item>
+            <item value="NX_ANGLE"></item>
+          </enumeration>
+        </attribute>
+      </field>
+
+      <field name="radial_axis_edges" type="NX_NUMBER" optional="true">
+        <dimensions rank="1">
+          <dim index="1" value="nRadEdge" />
+        </dimensions>
+        <attribute name="long_name" type="NX_CHAR">
+          <enumeration>
+            <item value="q bin edges"></item>
+            <item value="2theta bin edges"></item>
+          </enumeration>
+        </attribute>
+        <attribute name="units" type="NX_CHAR">
+          <enumeration>
+            <item value="NX_PER_LENGTH"></item>
+            <item value="NX_WAVENUMBER"></item>
+            <item value="NX_ANGLE"></item>
+          </enumeration>
+        </attribute>
+      </field>
+
+      <field name="norm" type="NX_NUMBER" optional="true">
+        <dimensions rank="1">
+          <dim index="1" value="nRad" />
+        </dimensions>
+        <attribute name="long_name" type="NX_CHAR">
+          <enumeration>
+            <item value="number of pixels contributing to the corresponding bin"></item>
```

Comment:
When correcting for solid angle, is this the "number" of pixels?

Reply:
Unfortunately this is not so simple. The descriptions is simplified. Note that it must be short as it will appear mainly as x-axis label in plots. A more precise definition is with the `normalization_applied` field: _the effective number or weighted contribution of detector pixels to each integration bin_. But that is too long.

Reply:
One can suggest different short description. However we should keep in mind people can use plots with such description for non-expert audience and we cannot explain all details how a particular sw is implementing all geometrical, mathematical and physical corrections that may be part of the normalization.